integ/src/updater.rs

```diff
@@ -158,6 +160,31 @@ pub async fn delete_cluster_resources(kube_config_path: &str) -> UpdaterResult<(
     Ok(())
 }
 
+// create/delete statefulset pods, stateless nginx pods, and pods with PDBs on EKS cluster
+pub async fn process_pods_test(action: &str, kube_config_path: &str) -> UpdaterResult<()> {
+    let pods_status = Command::new("kubectl")
```

Comment:
nit: I've seen that we usually refer to `<binary_name>` with the full path to the binary, also, this should be a constant.